kraken/spot/user/__init__.py

```diff
@@ -84,6 +84,20 @@ def get_account_balance(self: "User") -> dict:
             method="POST", uri="/private/Balance"
         )
 
+    def get_balances_and_excluded(self: "User") -> dict:
```

Comment:
The naming might be better, even its a breaking change.

```suggestion
    def get_balances(self: "User") -> dict:
```